proc/groupby.go

```diff
@@ -25,11 +25,11 @@ type GroupByKey struct {
 }
 
 type GroupByParams struct {
-	duration ast.Duration
-	limit    int
-	keys     []GroupByKey
-	reducers []compile.CompiledReducer
-	builder  *ColumnBuilder
+	limit        int
+	keys         []GroupByKey
+	reducers     []compile.CompiledReducer
+	builder      *ColumnBuilder
+	inputSortDir int
```

Comment:
Keep the configuration fields together and (mostly) sorted.
```suggestion
	inputSortDir int
	limit        int
	keys         []GroupByKey
	reducers     []compile.CompiledReducer
	builder      *ColumnBuilder
```